src/app/src/_presentation/pages/profile/projects/projects-page.component.ts

```diff
@@ -17,7 +16,6 @@ export class ProjectsPageComponent {
     this.projectRepository.getOrders()
       .pipe(finalize(() => this.didLoaded = true))
       .subscribe((items: Array<ProjectResponse>) => {
-        console.log(items);
         this.projects = items;
       });
```

Comment:
Remove redundant Brackets:

.subscribe((items: Array<ProjectResponse>) => this.projects = items);